test/run

```diff
@@ -158,6 +158,27 @@ function test_from_dockerfile_s2i() {
   test_from_dockerfile ".s2i"
 }
 
+function test_from_dockerfile_ruby_ex() {
+  dockerfile="Dockerfile${1:-}"
+  TESTCASE_RESULT=0
+  info "Check building using a $dockerfile on ruby-ex repository."
+
+  # Ruby 3.3 introduced too many incompatibilities to be able
+  # to use the same Gemfile for RHEL 7 and also newer RHELs.
+  # we can use the same Gemfile for RHEL 7 and newer
+  # as long as Ruby MAJOR.MINOR <= 3.1. Newer Ruby needs dependencies
+  # that are not compatible with RHEL 7.
+
```

Comment:
comment is probably not needed for ruby-ex so far ;)

Reply:
Fixed by https://github.com/sclorg/s2i-ruby-container/pull/570/commits/d8e7390fe222a167bff05b190358917b39391944